docs/reference/changelog-r2019.md

```diff
@@ -15,6 +15,7 @@ Released on ???
     - Upgraded to Qt 5.13.1 on Windows and macOS.
   - Bug fixes
     - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
+    - Fixed [infra-red DistanceSensors](distancesensor.md) vs [Plane](plane.md) collision detection.
```

Comment:
Note that it may influence the PEN device too.

Reply:
Very true, I added it in the change log.